backend/src/test/java/com/bakdata/conquery/models/datasets/concepts/filters/TestGroupFilter.java

```diff
@@ -0,0 +1,90 @@
+package com.bakdata.conquery.models.datasets.concepts.filters;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+
+import com.bakdata.conquery.apiv1.frontend.FEFilter;
+import com.bakdata.conquery.apiv1.frontend.FEFilterType;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.io.jackson.InternalOnly;
+import com.bakdata.conquery.models.datasets.Column;
+import com.bakdata.conquery.models.datasets.concepts.filters.specific.QueryContextResolvable;
+import com.bakdata.conquery.models.query.QueryResolveContext;
+import com.bakdata.conquery.models.query.filter.event.MultiSelectFilterNode;
+import com.bakdata.conquery.models.query.queryplan.filter.FilterNode;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.experimental.FieldNameConstants;
+
+@CPSType(id = "TEST_GROUP_FILTER", base = Filter.class)
+public class TestGroupFilter extends SingleColumnFilter<TestGroupFilter.GroupFilterValue> implements GroupFilter {
```

Comment:
Würde glaube ich in den Docs von GroupValue auf die Klasse als Beispiel verweisen